webrtc-stats.html

```diff
@@ -142,11 +142,12 @@
             </p>
```

Comment:
If session exists, it should point to RTCPeerConnectionStats. But should we rename it to peer-connection?